qlib/data/dataset/handler.py

```diff
@@ -111,16 +112,17 @@ def conf_data(self, **kwargs):
         The data will be initialized with different time range.
 
         """
-        attr_list = {"instruments", "start_time", "end_time"}
-        for k, v in kwargs.items():
-            if k in attr_list:
-                setattr(self, k, v)
-            else:
-                raise KeyError("Such config is not supported.")
-
-    def init(self, enable_cache: bool = False):
+        super().config(**kwargs)
+        if instruments:
+            self.instruments = instruments
+        if start_time:
```

Comment:
`**kwargs` should be used to tell if user are config `start_time` to None or   don't want to config the setting.